functions/src/common/auth.ts

```diff
@@ -14,13 +14,16 @@
  * limitations under the License.
  */
 
-import {DecodedIdToken, getAuth} from 'firebase-admin/auth';
-import {DocumentSnapshot} from 'firebase-admin/firestore';
-import {https, Response} from 'firebase-functions/v1';
+import { DecodedIdToken, getAuth } from 'firebase-admin/auth';
+import { DocumentSnapshot } from 'firebase-admin/firestore';
+import { https, Response } from 'firebase-functions/v1';
+import { EmulatorIdToken } from '../handlers';
```

Comment:
You may also need to remove the space between brackets.

Reply:
Done.